src/linear_algebra/affine_space/affine_subspace.lean

```diff
@@ -1377,3 +1377,85 @@ lemma comap_supr {ι : Sort*} (f : P₁ →ᵃ[k] P₂) (s : ι → affine_subsp
 end affine_subspace
 
 end map_comap
+
+namespace affine_subspace
+
+open affine_equiv
+
+variables {k : Type*} {V : Type*} {P : Type*} [ring k] [add_comm_group V] [module k V]
+variables [affine_space V P]
+include V
+
+/-- Two affine subspaces are parallel if one is related to the other by adding the same vector
+to all points. -/
+def parallel (s₁ s₂ : affine_subspace k P) : Prop :=
```

Comment:
I think it's worth mentioning this definition and the notation in the module docstring at the top of the file.

Reply:
Done.